src/Redux/actionCreators/UserActions.js

```diff
@@ -157,6 +158,7 @@ const fetchUserDetailsError = err => dispatch => {
 export const fetchUserDetails = async dispatch => {
   dispatch(loaderActions.startAppLoader(LoaderContent.APP_INIT));
   try {
+    dispatch(fetchUSDConversionRate);
```

Comment:
We should move this out of user details. Also we should await for this information so that we can be sure we have the conversion rates when we want to use it.

Reply:
Made changes for this comment. please have a look https://github.com/singnet/snet-dapp/pull/472/commits/5579b182c5e7f1d17541d214e134052c3f5ae541